templates/list.jsx

```diff
@@ -33,7 +33,7 @@ export default function List({ _columns, _orderedList, _items, ...props }) {
                 _classes
               ])}
               role="listitem"
-              style={(hasColumns && Adapt.device.screenSize === 'large' && { width: `${100 / _columns}%` }) || null}
+              style={(hasColumns && Adapt.device.screenSize === 'medium' && { width: `${100 / _columns}%` }) || null}
```

Comment:
Needs to do medium and large.
Adapt.device is deprecated, please `import device from 'core/js/device';` and use `device` directly.

```suggestion
              style={(hasColumns && device.isScreenSizeMin('medium') && { width: `${100 / _columns}%` }) || null}
```

Reply:
@oliverfoster Do you mean, does the breakpoint need to be configurable? I did think of that but it just needs to switch at the mobile view really.

Reply:
No. I mean you need to change the code.

`{ width: "${100 / _columns}%" }` was applied at `large`, the maximum breakpoint when the code was out of alignment.
It should now apply from `medium` through `large` and `xlarge`.

Doing a string comparison to just medium `Adapt.device.screenSize === 'medium'` doesn't update the behaviour correctly to move it to the new 1 to 2 component layout boundary/

`isScreenSizeMin` is specifically for doing `>=medium`, the mobile-first javascript equivalent of `min-width`.

Reply:
@oliverfoster of course! Wow wasn't thinking there.